docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2820` Users will now be able to select if they want to view graphs based at 0 instead of the minimum in the selected period.
```

Comment:
```suggestion
* :feature:`2820` Users will now be able to select if they want to view graphs based at a 0 y-axis start instead of the minimum in the selected period.
```